apps/site/components/Common/Select/index.tsx

```diff
@@ -66,13 +68,32 @@ const Select: FC<SelectProps> = ({
     return mappedValues;
   }, [values]);
 
+  useEffect(() => {
```

Comment:
This sounds like a very inefficient way of solving an issue that could be solved only with CSS.